src/main/java/com/elastic/support/BaseInputs.java

```diff
@@ -128,6 +145,16 @@ protected void runOutputDirInteractive(){
 
     }
 
+    public List<String> validateArchiveType(String archiveType) {
+        for (ArchiveType type : ArchiveType.values()) {
+            if (type.name().equalsIgnoreCase(archiveType)) {
+                return null;
+            }
+        }
+
+        return Collections.singletonList("Provided archive type" + archiveType + " is not supported.");
```

Comment:
Nitpick improvement for readability:

```suggestion
        return Collections.singletonList("Provided archive type [" + archiveType + "] is not supported.");
```